api/user/userService.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+var User = require('./userModel').User;
+var constants = require('../constants/constants');
+
+exports.friendRequestService = function (user, userFriend, friendsRequest) {
+
+
+
+    switch (friendsRequest) {
+        case constants.CONFIRM:
+            return confirmOrRejectUserRequest(user, userFriend, constants.REJECT);
```

Comment:
having conditional logic in method is bad practice and it smells bad, method should be written by (**S**OLID) Single responsibility